### ISSUE TITLE
Implementation of ARiA-2

### ISSUE THREAD

**Issue by NarendraPatwardhan** [achaiah/pywick#16]
Hi!

Thanks alot for the inclusion of ARiA in your library! The implementation of ARiA in the library is correct, however I feel for ARiA-2 there were substitution errors which led to an incorrect formula. If you were to substitute and reduce (for speedup) you end up getting the following form: ARiA2(x) = x*sigmoid(beta*x)**alpha which can be implemented as

`
class Aria2(nn.Module):
    """
    ARiA2 activation function, a special case of ARiA, for ARiA = f(x, 1, 0, 1, 1, b, 1/a)
    """

    def __init__(self, a=1.5, b = 1.):
        super(Aria2, self).__init__()
        self.alpha = a
        self.beta = b

    def forward(self, x):
        return x * torch.sigmoid(self.beta*x)**(self.alpha)
`

**Comment by achaiah** [comment]
Hi, thanks for your feedback! I incorporated your changes into the WIP branch.
